HealthAPILibrary/GiniHealthAPILibrary/Sources/GiniHealthAPILibrary/Documents/DefaultDocumentService.swift

```diff
@@ -221,3 +255,9 @@ public final class DefaultDocumentService: DefaultDocumentServiceProtocol {
         file(urlString: urlString, resourceHandler: sessionManager.download, completion: completion)
     }
 }
+
+extension DefaultDocumentService {
```

Comment:
[Clarification]: do we need to have it as a public enum or it's enough for internal access?